src/templates/doc-page.js

```diff
@@ -97,6 +139,11 @@ export const pageQuery = graphql`
           frontmatter {
             title
           }
+          parent {
+            ... on File {
+              relativeDirectory
+            }
+          }
```

Comment:
is there a reason we didn't use markdownRemark for this? I think you can get slugs, and it would only have gotten the markdown files, instead of anything else that might be in the directory.